pkg/scaling/executor/scale_jobs.go

```diff
@@ -65,6 +65,13 @@ func (e *scaleExecutor) RequestJobScale(ctx context.Context, scaledJob *kedav1al
 		logger.V(1).Info("No change in activity")
 	}
 
+	// if isPaused is not an empty string, pause scaling
+	IsPaused := "autoscaling.keda.sh/paused"
```

Comment:
would be nice to define the annotation as a constant here: https://github.com/kedacore/keda/blob/e8162483d700c33da1fc4ea54b29a2d32b368469/controllers/keda/util/predicate.go instead of just string.